tremor-common/src/ids.rs

```diff
@@ -15,131 +15,63 @@
 // common id handling
 
 /// we namespace onramp, offramp and operator ids differently in order to avoid clashes
-const ONRAMP_ID_BASE: u64 = 0b0;
-const OPERATOR_ID_BASE: u64 = 0b1000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_u64;
-const OFFRAMP_ID_BASE: u64 = 0b1000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_u64;
+const ONRAMP_ID_BASE: u64 =    0b0;
+const OPERATOR_ID_BASE: u64 =  0b1000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_u64;
+const OFFRAMP_ID_BASE: u64 =   0b1000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_u64;
+const CONNECTOR_ID_BASE: u64 = 0b1000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_u64;
```

Comment:
looking at those, in hindsight, would it make sense to use the first 4 bit as type identifier and the rest as counter that way the values would become masks instead of base ID's (not that it'd change much bit it kind of feels cleaner looking at it now)
```suggestion
const ONRAMP_ID_BASE: u64 =    0x10_00_00_00_00_00_00_00_u64;
const OPERATOR_ID_BASE: u64 =  0x20_00_00_00_00_00_00_00_u64;
const OFFRAMP_ID_BASE: u64 =   0x30_00_00_00_00_00_00_00_u64;
const CONNECTOR_ID_BASE: u64 = 0x40_00_00_00_00_00_00_00_u64